### DISCUSSION TITLE
Difference between --localconfig and --config

### DISCUSSION THREAD

**Discussion by yi0n** [keepassxreboot/keepassxc#6754] [Q&A] [answered]
Hi,

the cmd options here https://keepassxc.org/docs/KeePassXC_UserGuide.html#_command_line_options listening a --config and --localconfig option.
I'm confused by this.
Do work both like this:
--config: Is like a global config-file for all users and
--localconfig: Is like a local one, which overrides settings from the global config file?

For e.g.:
Me and some friends share the portalbe KeepassXC on a network drive.
On that network drive is the keepassxc.ini.
Each person has on his computer a local keepassxc.ini
Everyone has a shortcut/link on his desktop to the portable app on the network drive as follows: N:/KeepassXCPortable/keepassxc.exe --config N:/KeepassXCPortable/config/keepassxc.ini --localconfig /%appdata%/Roaming/keepassxc/config/keepassxc.ini

Does it work like that?

Thanks!

**Comment by droidmonkey** [comment, marked as answer]
Local config is for data that is relevant to your specific machine only like last used database path, window size, etc. All other settings are stored in config. 

If you are sharing an instance of keepassxc just delete the ".portable" file in the main directory to have the config files stored in their default non-portable locations.
